packages/@react-aria/toast/src/useToastRegion.ts

```diff
@@ -38,10 +39,20 @@ export function useToastRegion<T>(props: AriaToastRegionProps, state: ToastState
   });
 
   let lastFocused = useRef(null);
+  let timeoutId:ReturnType<typeof setTimeout>;
   let {focusWithinProps} = useFocusWithin({
     onFocusWithin: (e) => {
       state.pauseAll();
       lastFocused.current = e.relatedTarget;
+      if (ref.current?.contains(document.activeElement)) {
+        let count: number = state.visibleToasts.length || 0;
+        let announcement = stringFormatter.format('countAnnouncement', {count});
+        if (timeoutId) {
+          clearTimeout(timeoutId);
+          timeoutId = undefined;
+        }
+        timeoutId = setTimeout(() => announce(announcement, 'polite'), 750);
```

Comment:
where'd this timeout value from from?

Reply:
Probably trial and error in an effort to allow ToastRegion to announce on focus before the number of notifications within it.